app/controllers/stay/admin/bookings_controller.rb

```diff
@@ -4,18 +4,18 @@ class BookingsController < ApplicationController
       before_action :set_booking, only: %i[show edit update destroy]
 
       def index
-        @bookings = Stay::Booking.all
+        @bookings = current_user.bookings.all
```

Comment:
I think Booking is in vendor level . means let one hotel can multiple booking so its not nessesary that  booking is associated with that current user . and one booking is associate with one user only .